HerPublicWebsite.BusinessLogic/ExternalServices/EpbEpc/EpbEpcApi.cs

```diff
@@ -1,11 +1,80 @@
-﻿namespace HerPublicWebsite.BusinessLogic.ExternalServices.EpbEpc
+﻿using HerPublicWebsite.BusinessLogic.Models.Enums;
+using HerPublicWebsite.BusinessLogic.ExternalServices.Common;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System.Net.Http.Headers;
+
+namespace HerPublicWebsite.BusinessLogic.ExternalServices.EpbEpc
 {
+
+    public record struct EpcAddress
+    {
+        [JsonProperty("addressLine1")]
+        public string AddressLine1 { get; set; }
+
+        [JsonProperty("addressLine2")]
+        public string AddressLine2 { get; set; }
+
+        [JsonProperty("addressLine3")]
+        public string AddressLine3 { get; set; }
+
+        [JsonProperty("addressLine4")]
+        public string AddressLine4 { get; set; }
+
+        [JsonProperty("town")]
+        public string Town { get; set; }
+
+        [JsonProperty("postcode")]
+        public string Postcode { get; set; }
+    }
+
+    public record struct EpcAssessment
+    {
+        [JsonProperty("address")]
+        public EpcAddress Address { get; set; }
+
+        [JsonProperty("uprn")]
+        public string Uprn { get; set; }
+
+        [JsonProperty("lodgementDate")]
+        public DateTime LodgementDate { get; set; }
+
+        [JsonProperty("currentBand")]
+        public EpcRating CurrentBand { get; set; }
+    }
+
+    public record struct Data
+    {
+        [JsonProperty("assessment")]
+        public EpcAssessment Assessment { get; set; }
+    }
+
+    public record struct Root
```

Comment:
There's no Root class in the SEA code, so I suspect we won't need this here.

Reply:
Ah, no, you were right. We do need one, it's just called something else in SEA. Good rename here thanks